wagtail/admin/templates/wagtailadmin/shared/dialog/dialog.html

```diff
@@ -0,0 +1,45 @@
+{% load wagtailadmin_tags i18n %}
+{% with id=title|cautious_slugify %}
+    <template data-dialog>
+        <div
+            id="{{ id }}"
+            aria-labelledby="title-{{ id }}"
+            aria-hidden="true"
+            class="{{ class_name }} dialog"
```

Comment:
I’d prefer we don’t support arbitrary classes. If we want to create variations of the dialog component then they should have a proper API rather than relying on class names.

```suggestion
            class="dialog"
```